[Automation] native-image run fails for org.apache.activemq:activemq-broker:6.0.0

## native-image run on ubuntu-latest / JDK latest-ea / mode current-defaults

Failure kind: native-image run
OS: ubuntu-latest
JDK: latest-ea
Native-image mode: current-defaults
Reproducer: `GVM_TCK_NATIVE_IMAGE_MODE="current-defaults" GVM_TCK_LV="6.0.0" ./gradlew clean test -Pcoordinates="org.apache.activemq:activemq-broker:6.0.0"`
Runner log: https://github.com/oracle/graalvm-reachability-metadata/actions/runs/25727047580/job/75542581032
Last 300 lines of the log (excerpt truncated to fit the GitHub issue size limit):
```console
ports=jdk.internal.vm.ci/jdk.vm.ci.runtime=com.oracle.graal.graal_enterprise,jdk.graal.compiler,jdk.graal.compiler.hostvmaccess,org.graalvm.nativeimage.base,org.graalvm.nativeimage.builder,org.graalvm.nativeimage.pointsto \
--add-exports=jdk.internal.vm.ci/jdk.vm.ci.services=com.oracle.graal.graal_enterprise,com.oracle.svm.svm_enterprise,jdk.graal.compiler,org.graalvm.nativeimage.builder \
--add-exports=jdk.jfr/jdk.jfr.events=org.graalvm.nativeimage.builder \
--add-exports=jdk.jfr/jdk.jfr.internal.event=org.graalvm.nativeimage.builder \
--add-exports=jdk.jfr/jdk.jfr.internal.jfc=com.oracle.svm.svm_enterprise,org.graalvm.nativeimage.builder \
--add-exports=jdk.jfr/jdk.jfr.internal.settings=org.graalvm.nativeimage.builder \
--add-exports=jdk.jfr/jdk.jfr.internal.tracing=org.graalvm.nativeimage.builder \
--add-exports=jdk.jfr/jdk.jfr.internal=com.oracle.svm.svm_enterprise,org.graalvm.nativeimage.builder \
--add-exports=jdk.management/com.sun.management.internal=org.graalvm.nativeimage.builder \
-XX:+UseJVMCINativeLibrary \
-XX:MaxRAMPercentage=84.99999999761413 \
'-Dsvm.build.memoryUsageReasonText=13.27GiB of memory (85% of system memory because $CI set to '"'"'true'"'"')' \
-Xss10m \
-XX:+UseParallelGC \
-XX:GCTimeRatio=9 \
-XX:+ExitOnOutOfMemoryError \
-XX:+UseCompressedOops \
-Djava.awt.headless=true \
'-Dorg.graalvm.vendor=Oracle Corporation' \
-Dorg.graalvm.vendorurl=https://www.graalvm.org/ \
'-Dorg.graalvm.vendorversion=Oracle GraalVM 25.1.0-dev+10.1' \
-Dorg.graalvm.version=25.1.0-dev \
-Dcom.oracle.graalvm.isaot=true \
-Djava.system.class.loader=com.oracle.svm.hosted.NativeImageSystemClassLoader \
-Djava.util.concurrent.ForkJoinPool.common.exceptionHandler=com.oracle.svm.hosted.CommonPoolUncaughtExceptionHandler \
'-Djava.util.concurrent.ForkJoinPool.common.threadFactory=com.oracle.svm.hosted.NativeImageSystemClassLoader$NativeImageForkJoinWorkerThreadFactory' \
-Dorg.graalvm.nativeimage.imagecode=buildtime \
-Xshare:off \
-Djdk.internal.lambda.disableEagerInitialization=true \
-Djdk.internal.lambda.eagerlyInitialize=false \
-Djava.lang.invoke.InnerClassLambdaMetafactory.initializeLambdas=false \
-Djava.lang.invoke.MethodHandle.DONT_INLINE_THRESHOLD=-1 \
-Djava.lang.invoke.MethodHandle.PROFILE_GWT=false \
-Djunit.platform.listeners.uid.tracking.output.dir=/home/runner/work/graalvm-reachability-metadata/graalvm-reachability-metadata/tests/src/org.apache.activemq/activemq-broker/6.0.0/build/test-results/test/testlist \
-Dcom.oracle.svm.provided=/opt/hostedtoolcache/graalvm-jdk-latest-ea_linux-x64_bin/25.0.0/x64/graalvm-25.1.0-dev+10.1/lib/svm/svm-guest-staging.jar:/opt/hostedtoolcache/graalvm-jdk-latest-ea_linux-x64_bin/25.0.0/x64/graalvm-25.1.0-dev+10.1/lib/svm/library-support.jar:/opt/hostedtoolcache/graalvm-jdk-latest-ea_linux-x64_bin/25.0.0/x64/graalvm-25.1.0-dev+10.1/lib/svm/svm-guest.jar:/opt/hostedtoolcache/graalvm-jdk-latest-ea_linux-x64_bin/25.0.0/x64/graalvm-25.1.0-dev+10.1/lib/svm/svm-shared.jar:/opt/hostedtoolcache/graalvm-jdk-latest-ea_linux-x64_bin/25.0.0/x64/graalvm-25.1.0-dev+10.1/lib/svm/vmaccess-guest.jar \
--add-modules=ALL-DEFAULT \
--enable-native-access=org.graalvm.nativeimage.guest.staging,org.graalvm.espresso.shared.svm,org.graalvm.nativeimage.objectfile,org.graalvm.nativeimage.shared,org.graalvm.nativeimage.shaded.capnproto,org.graalvm.nativeimage.foreign,com.oracle.graal.reporter,org.graalvm.nativeimage.builder,jdk.graal.compiler.vmaccess.guest,jdk.graal.compiler.hostvmaccess,jdk.graal.compiler.vmaccess,com.oracle.svm.svm_enterprise,org.graalvm.nativeimage.base,org.graalvm.nativeimage.configure,org.graalvm.nativeimage.pointsto \
--module-path \
/opt/hostedtoolcache/graalvm-jdk-latest-ea_linux-x64_bin/25.0.0/x64/graalvm-25.1.0-dev+10.1/lib/svm/builder/hostvmaccess.jar:/opt/hostedtoolcache/graalvm-jdk-latest-ea_linux-x64_bin/25.0.0/x64/graalvm-25.1.0-dev+10.1/lib/svm/builder/svm-guest-staging.jar:/opt/hostedtoolcache/graalvm-jdk-latest-ea_linux-x64_bin/25.0.0/x64/graalvm-25.1.0-dev+10.1/lib/svm/builder/reporter.jar:/opt/hostedtoolcache/graalvm-jdk-latest-ea_linux-x64_bin/25.0.0/x64/graalvm-25.1.0-dev+10.1/lib/svm/builder/objectfile.jar:/opt/hostedtoolcache/graalvm-jdk-latest-ea_linux-x64_bin/25.0.0/x64/graalvm-25.1.0-dev+10.1/lib/svm/builder/svm-shared.jar:/opt/hostedtoolcache/graalvm-jdk-latest-ea_linux-x64_bin/25.0.0/x64/graalvm-25.1.0-dev+10.1/lib/svm/builder/vmaccess-guest.jar:/opt/hostedtoolcache/graalvm-jdk-latest-ea_linux-x64_bin/25.0.0/x64/graalvm-25.1.0-dev+10.1/lib/svm/builder/svm-foreign.jar:/opt/hostedtoolcache/graalvm-jdk-latest-ea_linux-x64_bin/25.0.0/x64/graalvm-25.1.0-dev+10.1/lib/svm/builder/vmaccess.jar:/opt/hostedtoolcache/graalvm-jdk-latest-ea_linux-x64_bin/25.0.0/x64/graalvm-25.1.0-dev+10.1/lib/svm/builder/pointsto.jar:/opt/hostedtoolcache/graalvm-jdk-latest-ea_linux-x64_bin/25.0.0/x64/graalvm-25.1.0-dev+10.1/lib/svm/builder/native-image-base.jar:/opt/hostedtoolcache/graalvm-jdk-latest-ea_linux-x64_bin/25.0.0/x64/graalvm-25.1.0-dev+10.1/lib/svm/builder/espresso-svm.jar:/opt/hostedtoolcache/graalvm-jdk-latest-ea_linux-x64_bin/25.0.0/x64/graalvm-25.1.0-dev+10.1/lib/svm/builder/svm.jar:/opt/hostedtoolcache/graalvm-jdk-latest-ea_linux-x64_bin/25.0.0/x64/graalvm-25.1.0-dev+10.1/lib/svm/builder/svm-capnproto-runtime.jar:/opt/hostedtoolcache/graalvm-jdk-latest-ea_linux-x64_bin/25.0.0/x64/graalvm-25.1.0-dev+10.1/lib/svm/builder/svm-enterprise.jar:/opt/hostedtoolcache/graalvm-jdk-latest-ea_linux-x64_bin/25.0.0/x64/graalvm-25.1.0-dev+10.1/lib/svm/builder/svm-configure.jar \
--module \
org.graalvm.nativeimage.builder/com.oracle.svm.hosted.NativeImageGeneratorRunner \
-keepalive \
/proc/12448/comm \
-imagecp \
/home/runner/work/graalvm-reachability-metadata/graalvm-reachability-metadata/tests/src/org.apache.activemq/activemq-broker/6.0.0/build/libs/org.apache.activemq.activemq-broker_tests.jar:/home/runner/.gradle/caches/modules-2/files-2.1/org.graalvm.buildtools/junit-platform-native/1.0.0/2974f834a2f0a0105f2284fba7ef16dc4d60de56/junit-platform-native-1.0.0.jar:/home/runner/.gradle/caches/modules-2/files-2.1/org.junit.jupiter/junit-jupiter/5.9.2/26c586fbe0ebd81b48c9f11f0d998124248697ae/junit-jupiter-5.9.2.jar:/home/runner/.gradle/caches/modules-2/files-2.1/org.junit.platform/junit-platform-reporting/1.9.2/8bea292e2a546e4c21071f6bee60917ffd9f0101/junit-platform-reporting-1.9.2.jar:/home/runner/.gradle/caches/modules-2/files-2.1/org.junit.platform/junit-platform-launcher/1.9.2/38e0bfad8c57d4cd1a8f27926c25ffe9543068d6/junit-platform-launcher-1.9.2.jar:/home/runner/.gradle/caches/modules-2/files-2.1/org.junit.vintage/junit-vintage-engine/5.9.2/53421816bde124a564a64ba005dcc0c8e66a9722/junit-vintage-engine-5.9.2.jar:/home/runner/.gradle/caches/modules-2/files-2.1/org.junit.jupiter/junit-jupiter-params/5.9.2/bc2765afb7b85b583c710dd259a11c6b8c39e912/junit-jupiter-params-5.9.2.jar:/home/runner/.gradle/caches/modules-2/files-2.1/org.junit.jupiter/junit-jupiter-engine/5.9.2/572f7a553b53f83ee59cc045ce1c3772864ab76c/junit-jupiter-engine-5.9.2.jar:/home/runner/.gradle/caches/modules-2/files-2.1/org.junit.jupiter/junit-jupiter-api/5.9.2/fed843581520eac594bc36bb4b0f55e7b947dda9/junit-jupiter-api-5.9.2.jar:/home/runner/.gradle/caches/modules-2/files-2.1/org.junit.platform/junit-platform-engine/1.9.2/40aeef2be7b04f96bb91e8b054affc28b7c7c935/junit-platform-engine-1.9.2.jar:/home/runner/.gradle/caches/modules-2/files-2.1/org.junit.platform/junit-platform-commons/1.9.2/6f9f8621d8230cd38aa42e58ccbc0c00569131ce/junit-platform-commons-1.9.2.jar:/home/runner/.gradle/caches/modules-2/files-2.1/org.junit.platform/junit-platform-console/1.9.2/5548148dd55e820c42bd817ab93c1eaa05ce37f6/junit-platform-console-1.9.2.jar:/home/runner/work/graalvm-reachability-metadata/graalvm-reachability-metadata/tests/tck-test-support/build/libs/tck-test-support-1.0.0-SNAPSHOT.jar:/home/runner/.gradle/caches/modules-2/files-2.1/org.apache.activemq/activemq-broker/6.0.0/9e02abbcd317f4c0b22c96d29f63da2e0f7b9806/activemq-broker-6.0.0.jar:/home/runner/.gradle/caches/modules-2/files-2.1/org.assertj/assertj-core/3.22.0/c300c0c6a24559f35fa0bd3a5472dc1edcd0111e/assertj-core-3.22.0.jar:/home/runner/.gradle/caches/modules-2/files-2.1/org.apache.activemq/activemq-openwire-legacy/6.0.0/817a51aaaef75076d39aae18e5fff6f63f1e3c3b/activemq-openwire-legacy-6.0.0.jar:/home/runner/.gradle/caches/modules-2/files-2.1/org.apache.activemq/activemq-client/6.0.0/b4196c428e00601d9dd8a4c347a7117618a805eb/activemq-client-6.0.0.jar:/home/runner/.gradle/caches/modules-2/files-2.1/jakarta.annotation/jakarta.annotation-api/2.1.1/48b9bda22b091b1f48b13af03fe36db3be6e1ae3/jakarta.annotation-api-2.1.1.jar:/home/runner/.gradle/caches/modules-2/files-2.1/com.fasterxml.jackson.core/jackson-annotations/2.15.3/79baf4e605eb3bbb60b1c475d44a7aecceea1d60/jackson-annotations-2.15.3.jar:/home/runner/.gradle/caches/modules-2/files-2.1/com.fasterxml.jackson.core/jackson-core/2.15.3/60d600567c1862840397bf9ff5a92398edc5797b/jackson-core-2.15.3.jar:/home/runner/.gradle/caches/modules-2/files-2.1/com.fasterxml.jackson.core/jackson-databind/2.15.3/a734bc2c47a9453c4efa772461a3aeb273c010d9/jackson-databind-2.15.3.jar:/home/runner/.gradle/caches/modules-2/files-2.1/junit/junit/4.13.2/8ac9e16d933b6fb43bc7f576336b8f4d7eb5ba12/junit-4.13.2.jar:/home/runner/.gradle/caches/modules-2/files-2.1/org.slf4j/slf4j-api/2.0.9/7cf2726fdcfbc8610f9a71fb3ed639871f315340/slf4j-api-2.0.9.jar:/home/runner/.gradle/caches/modules-2/files-2.1/jakarta.jms/jakarta.jms-api/3.1.0/e194cf91a3f908e4846542849ac11a8e0b3c68ad/jakarta.jms-api-3.1.0.jar:/home/runner/.gradle/caches/modules-2/files-2.1/org.fusesource.hawtbuf/hawtbuf/1.11/8f0e50ad8bea37b84b698ec40cce09e47714a63e/hawtbuf-1.11.jar:/home/runner/.gradle/caches/modules-2/files-2.1/org.opentest4j/opentest4j/1.2.0/28c11eb91f9b6d8e200631d46e20a7f407f2a046/opentest4j-1.2.0.jar:/home/runner/.gradle/caches/modules-2/files-2.1/org.hamcrest/hamcrest-core/1.3/42a25dc3219429f0e5d060061f71acb49bf010a0/hamcrest-core-1.3.jar:/home/runner/work/graalvm-reachability-metadata/graalvm-reachability-metadata/tests/src/org.apache.activemq/activemq-broker/6.0.0/build/classes/java/test:/home/runner/work/graalvm-reachability-metadata/graalvm-reachability-metadata/tests/src/org.apache.activemq/activemq-broker/6.0.0/build/resources/test:/home/runner/work/graalvm-reachability-metadata/graalvm-reachability-metadata/tests/src/org.apache.activemq/activemq-broker/6.0.0/build/test-results/test/testlist \
-imagemp \
/opt/hostedtoolcache/graalvm-jdk-latest-ea_linux-x64_bin/25.0.0/x64/graalvm-25.1.0-dev+10.1/lib/svm/library-support.jar:/opt/hostedtoolcache/graalvm-jdk-latest-ea_linux-x64_bin/25.0.0/x64/graalvm-25.1.0-dev+10.1/lib/svm/svm-guest.jar \
-H:CLibraryPath=/opt/hostedtoolcache/graalvm-jdk-latest-ea_linux-x64_bin/25.0.0/x64/graalvm-25.1.0-dev+10.1/lib/svm/clibraries/linux-amd64/glibc,/opt/hostedtoolcache/graalvm-jdk-latest-ea_linux-x64_bin/25.0.0/x64/graalvm-25.1.0-dev+10.1/lib/svm/clibraries/linux-amd64,/opt/hostedtoolcache/graalvm-jdk-latest-ea_linux-x64_bin/25.0.0/x64/graalvm-25.1.0-dev+10.1/lib/svm/clibraries \
-H:Path@driver=/home/runner/work/graalvm-reachability-metadata/graalvm-reachability-metadata/tests/src/org.apache.activemq/activemq-broker/6.0.0/build/native/nativeTestCompile \
-H:BuildOutputJSONFile@user=/tmp/native-image-build-output.json \
-H:FallbackThreshold@user+api=0 \
-H:Name@user+api=org.apache.activemq.activemq-broker_tests-tests \
-H:ConfigurationFileDirectories@user=/home/runner/work/graalvm-reachability-metadata/graalvm-reachability-metadata/tests/src/org.apache.activemq/activemq-broker/6.0.0/build/native/generated/generateTestResourcesConfigFile,/home/runner/work/graalvm-reachability-metadata/graalvm-reachability-metadata/metadata/org.opentest4j/opentest4j/1.2.0,/home/runner/work/graalvm-reachability-metadata/graalvm-reachability-metadata/metadata/org.junit.platform/junit-platform-commons/1.8.2,/home/runner/work/graalvm-reachability-metadata/graalvm-reachability-metadata/metadata/org.junit.jupiter/junit-jupiter-params/5.9.2,/home/runner/work/graalvm-reachability-metadata/graalvm-reachability-metadata/metadata/org.junit.jupiter/junit-jupiter-engine/5.8.2,/home/runner/work/graalvm-reachability-metadata/graalvm-reachability-metadata/metadata/org.junit.platform/junit-platform-engine/1.10.4,/home/runner/work/graalvm-reachability-metadata/graalvm-reachability-metadata/metadata/org.junit.platform/junit-platform-reporting/1.9.2,/home/runner/work/graalvm-reachability-metadata/graalvm-reachability-metadata/metadata/org.junit.platform/junit-platform-launcher/1.9.2,/home/runner/work/graalvm-reachability-metadata/graalvm-reachability-metadata/metadata/org.junit.vintage/junit-vintage-engine/5.9.1,/home/runner/work/graalvm-reachability-metadata/graalvm-reachability-metadata/metadata/junit/junit/4.13.2,/home/runner/work/graalvm-reachability-metadata/graalvm-reachability-metadata/metadata/org.hamcrest/hamcrest-core/1.3,/home/runner/work/graalvm-reachability-metadata/graalvm-reachability-metadata/metadata/org.apache.activemq/activemq-broker/6.0.0,/home/runner/work/graalvm-reachability-metadata/graalvm-reachability-metadata/metadata/org.apache.activemq/activemq-client/6.0.0,/home/runner/work/graalvm-reachability-metadata/graalvm-reachability-metadata/metadata/org.slf4j/slf4j-api/1.7.36,/home/runner/work/graalvm-reachability-metadata/graalvm-reachability-metadata/metadata/jakarta.jms/jakarta.jms-api/3.0.0,/home/runner/work/graalvm-reachability-metadata/graalvm-reachability-metadata/metadata/org.fusesource.hawtbuf/hawtbuf/1.11,/home/runner/work/graalvm-reachability-metadata/graalvm-reachability-metadata/metadata/org.apache.activemq/activemq-openwire-legacy/5.18.1,/home/runner/work/graalvm-reachability-metadata/graalvm-reachability-metadata/metadata/jakarta.annotation/jakarta.annotation-api/2.1.0,/home/runner/work/graalvm-reachability-metadata/graalvm-reachability-metadata/metadata/com.fasterxml.jackson.core/jackson-databind/2.15.2,/home/runner/work/graalvm-reachability-metadata/graalvm-reachability-metadata/metadata/com.fasterxml.jackson.core/jackson-annotations/2.3.0,/home/runner/work/graalvm-reachability-metadata/graalvm-reachability-metadata/metadata/com.fasterxml.jackson.core/jackson-core/2.6.0 \
-H:Optimize@user+api=b \
-H:+ReportExceptionStackTraces@user \
-H:Features@user+api=org.graalvm.junit.platform.JUnitPlatformFeature \
'-H:Class@explicit main-class=org.graalvm.junit.platform.NativeImageJUnitLauncher' \
-H:ImageBuildID@driver=36731a88-d9ce-578b-f81c-b00882537d9f \
'-H:Features@jar:file:///opt/hostedtoolcache/graalvm-jdk-latest-ea_linux-x64_bin/25.0.0/x64/graalvm-25.1.0-dev+10.1/lib/svm/library-support.jar!/META-INF/native-image/com.oracle.svm/thirdparty/native-image.properties+api=com.oracle.svm.thirdparty.gson.GsonFeature' \
'-H:Features@jar:file:///opt/hostedtoolcache/graalvm-jdk-latest-ea_linux-x64_bin/25.0.0/x64/graalvm-25.1.0-dev+10.1/lib/svm/library-support.jar!/META-INF/native-image/com.oracle.svm/polyglot/native-image.properties+api=com.oracle.svm.polyglot.groovy.GroovyIndyInterfaceFeature' \
'-H:AddExports@jar:file:///opt/hostedtoolcache/graalvm-jdk-latest-ea_linux-x64_bin/25.0.0/x64/graalvm-25.1.0-dev+10.1/lib/svm/svm-guest.jar!/META-INF/native-image/org.graalvm.nativeimage.guest/native-image.properties+api=java.base/jdk.internal.misc=org.graalvm.nativeimage.guest' \
'-H:Features@jar:file:///opt/hostedtoolcache/graalvm-jdk-latest-ea_linux-x64_bin/25.0.0/x64/graalvm-25.1.0-dev+10.1/lib/svm/svm-guest.jar!/META-INF/native-image/org.graalvm.nativeimage.guest/native-image.properties+api=com.oracle.svm.guest.features.jdk.hosted.JavaNetHttpFeature' \
'-H:ClassInitialization@jar:file:///opt/hostedtoolcache/graalvm-jdk-latest-ea_linux-x64_bin/25.0.0/x64/graalvm-25.1.0-dev+10.1/lib/svm/svm-guest.jar!/META-INF/native-image/org.graalvm.nativeimage.guest/native-image.properties+api=sun.net.httpserver.simpleserver:run_time' \
'-H:ReachabilityMetadataResources@jar:file:///opt/hostedtoolcache/graalvm-jdk-latest-ea_linux-x64_bin/25.0.0/x64/graalvm-25.1.0-dev+10.1/lib/svm/svm-guest.jar!/META-INF/native-image/org.graalvm.nativeimage.guest/reachability-metadata.json+api=META-INF/native-image/org.graalvm.nativeimage.guest/reachability-metadata.json' \
-H:DriverWarningsCount=4
]
Warning: Option 'FallbackThreshold' is deprecated and might be removed in a future release: It no longer has any effect, and no replacement is available. Please refer to the GraalVM release notes.
========================================================================================================================
GraalVM Native Image: Generating 'org.apache.activemq.activemq-broker_tests-tests' (executable)...
========================================================================================================================
For detailed information and explanations on the build output, visit:
https://github.com/oracle/graal/blob/master/docs/reference-manual/native-image/BuildOutput.md
------------------------------------------------------------------------------------------------------------------------
[1/8] Initializing...                                                                                   (6.8s @ 0.22GiB)
 Builder configuration:
 - Java version: 25.0.2+10-LTS, vendor version: Oracle GraalVM 25.1.0-dev+10.1
 - Graal compiler: optimization level: b, target machine: x86-64-v3, PGO: off
 - C compiler: gcc (linux, x86_64, 13.3.0)
 - Assertions: disabled, system assertions: disabled
 - 1 user-specific feature(s):
   - org.graalvm.junit.platform.JUnitPlatformFeature
 Image configuration:
 - Garbage collector: Serial GC (max heap size: 80% of RAM)
 - Assertions: disabled (class-specific config may apply), system assertions: disabled
------------------------------------------------------------------------------------------------------------------------
 1 experimental option(s) unlocked:
 - '-H:BuildOutputJSONFile' (origin(s): command line)
------------------------------------------------------------------------------------------------------------------------
Build resources:
 - 13.27GiB of memory (85% of system memory because $CI set to 'true')
 - 4 thread(s) (100.0% of 4 available processor(s), determined at start)
[junit-platform-native] Running in 'test listener' mode using files matching pattern [junit-platform-unique-ids*] found in folder [/home/runner/work/graalvm-reachability-metadata/graalvm-reachability-metadata/tests/src/org.apache.activemq/activemq-broker/6.0.0/build/test-results/test/testlist] and its subfolders.
[2/8] Performing analysis...  [*****]                                                                  (39.9s @ 1.06GiB)
   10,281 types,  12,207 fields, and  51,803 methods found reachable
      350 types,      89 fields, and     806 methods registered for reflection
       63 types,      71 fields, and      55 methods registered for JNI access
      157 resource accesses registered with 266.42KiB total size
        4 native libraries: dl, pthread, rt, z
[3/8] Building universe...                                                                              (5.9s @ 0.97GiB)
[4/8] Parsing methods...      [**]                                                                      (4.2s @ 1.05GiB)
[5/8] Inlining methods...     [***]                                                                     (2.3s @ 1.13GiB)
[6/8] Compiling methods...    [*****]                                                                  (24.6s @ 0.85GiB)
[7/8] Laying out methods...   [**]                                                                      (3.6s @ 1.09GiB)
[8/8] Creating image...       [**]                                                                      (3.6s @ 1.24GiB)
  12.65MiB (37.85%) for code area:    32,307 compilation units
  15.94MiB (47.69%) for image heap:  247,423 objects and 27 resources
   4.83MiB (14.46%) for other data
  33.42MiB in total image size, 30.69MiB in total file size
------------------------------------------------------------------------------------------------------------------------
Top 10 origins of code area:                                Top 10 object types in image heap:
   1.18MiB o.g.n.runtime/com.oracle.svm.core.code              3.96MiB byte[] for code metadata
 741.92KiB java.base/java.util                                 2.93MiB byte[] for string data
 408.33KiB java.base/jdk.internal.classfile.impl               1.58MiB java.base/java.lang.String
 340.37KiB java.base/java.lang                                 1.27MiB java.base/java.lang.Class
 325.64KiB org.apache.activemq.broker.region                   1.10MiB o.g.n.runtime/c.o.s.core.hub.DynamicHubCompanion
 275.98KiB java.base/java.util.concurrent                    597.64KiB byte[] for general heap data
 231.31KiB java.base/java.io                                 483.52KiB byte[] for reflection metadata
 229.36KiB java.base/java.text                               399.14KiB o.g.n.runtime/c.o.s.c.h.ImageReflectionMetadata
 224.55KiB java.base/com.sun.crypto.provider                 382.20KiB heap alignment
 223.43KiB org.assertj.core.internal.bytebuddy.jar.asm       308.28KiB java.base/java.util.HashMap$Node
   8.53MiB for 401 more packages                               2.97MiB for 2847 more object types
                            Use '--emit build-report' to create a report with more details.
------------------------------------------------------------------------------------------------------------------------
Security report:
 - Binary includes Java deserialization.
 - CycloneDX SBOM with 24 component(s) is embedded in binary (1.26KiB). 1 type(s) could not be associated to a component.
 - Advanced obfuscation not enabled; enable with '-H:AdvancedObfuscation=""' (experimental support).
------------------------------------------------------------------------------------------------------------------------
Recommendations:
 G1GC: Use the G1 GC ('--gc=G1') for improved latency and throughput.
 PGO:  Use Profile-Guided Optimizations ('--pgo') for improved throughput.
 FUTR: Use '--future-defaults=all' to prepare for future releases.
 HEAP: Set max heap for improved and more predictable memory usage.
 CPU:  Enable more CPU features with '-march=native' for improved performance.
------------------------------------------------------------------------------------------------------------------------
                       7.3s (7.7% of total time) in 915 GCs | Peak RSS: 1.93GiB | CPU load: 3.64
------------------------------------------------------------------------------------------------------------------------
Build artifacts:
 /home/runner/work/graalvm-reachability-metadata/graalvm-reachability-metadata/tests/src/org.apache.activemq/activemq-broker/6.0.0/build/native/nativeTestCompile/libmanagement_ext.so (jdk_library, 32.23KiB)
 /home/runner/work/graalvm-reachability-metadata/graalvm-reachability-metadata/tests/src/org.apache.activemq/activemq-broker/6.0.0/build/native/nativeTestCompile/org.apache.activemq.activemq-broker_tests-tests (executable, 30.69MiB)
 /tmp/native-image-build-output.json (build_info, 1.04KiB)
========================================================================================================================
Finished generating 'org.apache.activemq.activemq-broker_tests-tests' in 1m 33s.

The build process encountered 5 warnings.

[native-image-plugin] Native Image written to: /home/runner/work/graalvm-reachability-metadata/graalvm-reachability-metadata/tests/src/org.apache.activemq/activemq-broker/6.0.0/build/native/nativeTestCompile

> Task :nativeTest FAILED
SLF4J: No SLF4J providers were found.[junit-platform-native] WARNING: -djunit.platform.listeners.uid.tracking.output.dir not specified, trying to find test-ids on default Gradle/Maven locations. As this is a fallback mode, it could take a while. This should only happen if you are running tests executable manually and you didn't pass uid output directory with -djunit.platform.listeners.uid.tracking.output.dir=<path>.
[junit-platform-native] WARNING: Using test-ids from default Gradle project location:/home/runner/work/graalvm-reachability-metadata/graalvm-reachability-metadata/tests/src/org.apache.activemq/activemq-broker/6.0.0/build/test-results/test/testlist
[junit-platform-native] Running in 'test listener' mode using files matching pattern [junit-platform-unique-ids*] found in folder [/home/runner/work/graalvm-reachability-metadata/graalvm-reachability-metadata/tests/src/org.apache.activemq/activemq-broker/6.0.0/build/test-results/test/testlist] and its subfolders.
JUnit Platform on Native Image - report
----------------------------------------


SLF4J: Defaulting to no-operation (NOP) logger implementation
SLF4J: See https://www.slf4j.org/codes.html#noProviders for further details.

FAILURE: Build failed with an exception.

* What went wrong:
Execution failed for task ':nativeTest'.
> Process 'command '/home/runner/work/graalvm-reachability-metadata/graalvm-reachability-metadata/tests/src/org.apache.activemq/activemq-broker/6.0.0/build/native/nativeTestCompile/org.apache.activemq.activemq-broker_tests-tests'' finished with non-zero exit value 1

* Try:
> Run with --stacktrace option to get the stack trace.
> Run with --info or --debug option to get more log output.
> Run with --scan to generate a Build Scan (Powered by Develocity).
> Get more help at https://help.gradle.org.

BUILD FAILED in 1m 37s
org_apache_activemq.activemq_broker.ActivemqBrokerTest > testEmbeddedBrokerConnection() FAILED


Failures (1):
  JUnit Jupiter:ActivemqBrokerTest:testEmbeddedBrokerConnection()
    MethodSource [className = 'org_apache_activemq.activemq_broker.ActivemqBrokerTest', methodName = 'testEmbeddedBrokerConnection', methodParameterTypes = '']
    => jakarta.jms.JMSException: Could not create Transport. Reason: java.lang.NullPointerException
       org.apache.activemq.util.JMSExceptionSupport.create(JMSExceptionSupport.java:49)
       org.apache.activemq.ActiveMQConnectionFactory.createTransport(ActiveMQConnectionFactory.java:378)
       org.apache.activemq.ActiveMQConnectionFactory.createActiveMQConnection(ActiveMQConnectionFactory.java:391)
       org.apache.activemq.ActiveMQConnectionFactory.createActiveMQConnection(ActiveMQConnectionFactory.java:349)
       org.apache.activemq.ActiveMQConnectionFactory.createConnection(ActiveMQConnectionFactory.java:245)
       [...]
     Caused by: java.lang.NullPointerException
       org.apache.activemq.broker.jmx.ManagementContext.registerMBean(ManagementContext.java:415)
       org.apache.activemq.broker.jmx.AnnotatedMBean.registerMBean(AnnotatedMBean.java:93)
       org.apache.activemq.broker.BrokerService.startManagementContext(BrokerService.java:2612)
       org.apache.activemq.broker.BrokerService.start(BrokerService.java:607)
       org.apache.activemq.transport.vm.VMTransportFactory.doCompositeConnect(VMTransportFactory.java:127)
       [...]

Test run finished after 103 ms
[         3 containers found      ]
[         0 containers skipped    ]
[         3 containers started    ]
[         0 containers aborted    ]
[         3 containers successful ]
[         0 containers failed     ]
[         1 tests found           ]
[         0 tests skipped         ]
[         1 tests started         ]
[         0 tests aborted         ]
[         0 tests successful      ]
[         1 tests failed          ]


[Incubating] Problems report is available at: file:///home/runner/work/graalvm-reachability-metadata/graalvm-reachability-metadata/tests/src/org.apache.activemq/activemq-broker/6.0.0/build/reports/problems/problems-report.html

Deprecated Gradle features were used in this build, making it incompatible with Gradle 10.

You can use '--warning-mode all' to show the individual deprecation warnings and determine if they come from your own scripts or plugins.

For more on this, please refer to https://docs.gradle.org/9.1.0/userguide/command_line_interface.html#sec:command_line_warnings in the Gradle documentation.
17 actionable tasks: 7 executed, 10 up-to-date

> Task :tckTest FAILED

[Incubating] Problems report is available at: file:///home/runner/work/graalvm-reachability-metadata/graalvm-reachability-metadata/build/reports/problems/problems-report.html

FAILURE: Build failed with an exception.

* What went wrong:
Execution failed for task ':tckTest'.
> Test for org.apache.activemq:activemq-broker:6.0.0 failed with exit code 1.

* Try:
> Run with --stacktrace option to get the stack trace.
> Run with --info or --debug option to get more log output.
> Run with --scan to generate a Build Scan (Powered by Develocity).
> Get more help at https://help.gradle.org.

Deprecated Gradle features were used in this build, making it incompatible with Gradle 10.

You can use '--warning-mode all' to show the individual deprecation warnings and determine if they come from your own scripts or plugins.

For more on this, please refer to https://docs.gradle.org/9.1.0/userguide/command_line_interface.html#sec:command_line_warnings in the Gradle documentation.

BUILD FAILED in 1m 38s
11 actionable tasks: 3 executed, 8 up-to-date
Finished stage 'native-image run' for 6.0.0 at 2026-05-12T10:35:44Z with exit code 1 after 99s
❌ ERROR: 'native-image run' for 6.0.0 failed with exit code 1.
FAILED[native-image run][6.0.0][GVM_TCK_NATIVE_IMAGE_MODE="current-defaults" GVM_TCK_LV="6.0.0" ./gradlew clean test -Pcoordinates="org.apache.activemq:activemq-broker:6.0.0"]
```

## native-image run on ubuntu-latest / JDK latest-ea / mode future-defaults-all

Failure kind: native-image run
OS: ubuntu-latest
JDK: latest-ea
Native-image mode: future-defaults-all
Reproducer: `GVM_TCK_NATIVE_IMAGE_MODE="future-defaults-all" GVM_TCK_LV="6.0.0" ./gradlew clean test -Pcoordinates="org.apache.activemq:activemq-broker:6.0.0"`
Runner log: https://github.com/oracle/graalvm-reachability-metadata/actions/runs/25727047580/job/75542581025
Last 300 lines of the log (excerpt truncated to fit the GitHub issue size limit):
```console
exports=jdk.internal.vm.ci/jdk.vm.ci.runtime=com.oracle.graal.graal_enterprise,jdk.graal.compiler,jdk.graal.compiler.hostvmaccess,org.graalvm.nativeimage.base,org.graalvm.nativeimage.builder,org.graalvm.nativeimage.pointsto \
--add-exports=jdk.internal.vm.ci/jdk.vm.ci.services=com.oracle.graal.graal_enterprise,com.oracle.svm.svm_enterprise,jdk.graal.compiler,org.graalvm.nativeimage.builder \
--add-exports=jdk.jfr/jdk.jfr.events=org.graalvm.nativeimage.builder \
--add-exports=jdk.jfr/jdk.jfr.internal.event=org.graalvm.nativeimage.builder \
--add-exports=jdk.jfr/jdk.jfr.internal.jfc=com.oracle.svm.svm_enterprise,org.graalvm.nativeimage.builder \
--add-exports=jdk.jfr/jdk.jfr.internal.settings=org.graalvm.nativeimage.builder \
--add-exports=jdk.jfr/jdk.jfr.internal.tracing=org.graalvm.nativeimage.builder \
--add-exports=jdk.jfr/jdk.jfr.internal=com.oracle.svm.svm_enterprise,org.graalvm.nativeimage.builder \
--add-exports=jdk.management/com.sun.management.internal=org.graalvm.nativeimage.builder \
-XX:+UseJVMCINativeLibrary \
-XX:MaxRAMPercentage=85.0 \
'-Dsvm.build.memoryUsageReasonText=13.27GiB of memory (85% of system memory because $CI set to '"'"'true'"'"')' \
-Xss10m \
-XX:+UseParallelGC \
-XX:GCTimeRatio=9 \
-XX:+ExitOnOutOfMemoryError \
-XX:+UseCompressedOops \
-Djava.awt.headless=true \
'-Dorg.graalvm.vendor=Oracle Corporation' \
-Dorg.graalvm.vendorurl=https://www.graalvm.org/ \
'-Dorg.graalvm.vendorversion=Oracle GraalVM 25.1.0-dev+10.1' \
-Dorg.graalvm.version=25.1.0-dev \
-Dcom.oracle.graalvm.isaot=true \
-Djava.system.class.loader=com.oracle.svm.hosted.NativeImageSystemClassLoader \
-Djava.util.concurrent.ForkJoinPool.common.exceptionHandler=com.oracle.svm.hosted.CommonPoolUncaughtExceptionHandler \
'-Djava.util.concurrent.ForkJoinPool.common.threadFactory=com.oracle.svm.hosted.NativeImageSystemClassLoader$NativeImageForkJoinWorkerThreadFactory' \
-Dorg.graalvm.nativeimage.imagecode=buildtime \
-Xshare:off \
-Djdk.internal.lambda.disableEagerInitialization=true \
-Djdk.internal.lambda.eagerlyInitialize=false \
-Djava.lang.invoke.InnerClassLambdaMetafactory.initializeLambdas=false \
-Djava.lang.invoke.MethodHandle.DONT_INLINE_THRESHOLD=-1 \
-Djava.lang.invoke.MethodHandle.PROFILE_GWT=false \
-Djunit.platform.listeners.uid.tracking.output.dir=/home/runner/work/graalvm-reachability-metadata/graalvm-reachability-metadata/tests/src/org.apache.activemq/activemq-broker/6.0.0/build/test-results/test/testlist \
-Dcom.oracle.svm.provided=/opt/hostedtoolcache/graalvm-jdk-latest-ea_linux-x64_bin/25.0.0/x64/graalvm-25.1.0-dev+10.1/lib/svm/svm-guest-staging.jar:/opt/hostedtoolcache/graalvm-jdk-latest-ea_linux-x64_bin/25.0.0/x64/graalvm-25.1.0-dev+10.1/lib/svm/library-support.jar:/opt/hostedtoolcache/graalvm-jdk-latest-ea_linux-x64_bin/25.0.0/x64/graalvm-25.1.0-dev+10.1/lib/svm/svm-guest.jar:/opt/hostedtoolcache/graalvm-jdk-latest-ea_linux-x64_bin/25.0.0/x64/graalvm-25.1.0-dev+10.1/lib/svm/svm-shared.jar:/opt/hostedtoolcache/graalvm-jdk-latest-ea_linux-x64_bin/25.0.0/x64/graalvm-25.1.0-dev+10.1/lib/svm/vmaccess-guest.jar \
--add-modules=ALL-DEFAULT \
--enable-native-access=org.graalvm.nativeimage.guest.staging,org.graalvm.espresso.shared.svm,org.graalvm.nativeimage.objectfile,org.graalvm.nativeimage.shared,org.graalvm.nativeimage.shaded.capnproto,org.graalvm.nativeimage.foreign,com.oracle.graal.reporter,org.graalvm.nativeimage.builder,jdk.graal.compiler.vmaccess.guest,jdk.graal.compiler.hostvmaccess,jdk.graal.compiler.vmaccess,com.oracle.svm.svm_enterprise,org.graalvm.nativeimage.base,org.graalvm.nativeimage.configure,org.graalvm.nativeimage.pointsto \
--module-path \
/opt/hostedtoolcache/graalvm-jdk-latest-ea_linux-x64_bin/25.0.0/x64/graalvm-25.1.0-dev+10.1/lib/svm/builder/hostvmaccess.jar:/opt/hostedtoolcache/graalvm-jdk-latest-ea_linux-x64_bin/25.0.0/x64/graalvm-25.1.0-dev+10.1/lib/svm/builder/svm-guest-staging.jar:/opt/hostedtoolcache/graalvm-jdk-latest-ea_linux-x64_bin/25.0.0/x64/graalvm-25.1.0-dev+10.1/lib/svm/builder/reporter.jar:/opt/hostedtoolcache/graalvm-jdk-latest-ea_linux-x64_bin/25.0.0/x64/graalvm-25.1.0-dev+10.1/lib/svm/builder/objectfile.jar:/opt/hostedtoolcache/graalvm-jdk-latest-ea_linux-x64_bin/25.0.0/x64/graalvm-25.1.0-dev+10.1/lib/svm/builder/svm-shared.jar:/opt/hostedtoolcache/graalvm-jdk-latest-ea_linux-x64_bin/25.0.0/x64/graalvm-25.1.0-dev+10.1/lib/svm/builder/vmaccess-guest.jar:/opt/hostedtoolcache/graalvm-jdk-latest-ea_linux-x64_bin/25.0.0/x64/graalvm-25.1.0-dev+10.1/lib/svm/builder/svm-foreign.jar:/opt/hostedtoolcache/graalvm-jdk-latest-ea_linux-x64_bin/25.0.0/x64/graalvm-25.1.0-dev+10.1/lib/svm/builder/vmaccess.jar:/opt/hostedtoolcache/graalvm-jdk-latest-ea_linux-x64_bin/25.0.0/x64/graalvm-25.1.0-dev+10.1/lib/svm/builder/pointsto.jar:/opt/hostedtoolcache/graalvm-jdk-latest-ea_linux-x64_bin/25.0.0/x64/graalvm-25.1.0-dev+10.1/lib/svm/builder/native-image-base.jar:/opt/hostedtoolcache/graalvm-jdk-latest-ea_linux-x64_bin/25.0.0/x64/graalvm-25.1.0-dev+10.1/lib/svm/builder/espresso-svm.jar:/opt/hostedtoolcache/graalvm-jdk-latest-ea_linux-x64_bin/25.0.0/x64/graalvm-25.1.0-dev+10.1/lib/svm/builder/svm.jar:/opt/hostedtoolcache/graalvm-jdk-latest-ea_linux-x64_bin/25.0.0/x64/graalvm-25.1.0-dev+10.1/lib/svm/builder/svm-capnproto-runtime.jar:/opt/hostedtoolcache/graalvm-jdk-latest-ea_linux-x64_bin/25.0.0/x64/graalvm-25.1.0-dev+10.1/lib/svm/builder/svm-enterprise.jar:/opt/hostedtoolcache/graalvm-jdk-latest-ea_linux-x64_bin/25.0.0/x64/graalvm-25.1.0-dev+10.1/lib/svm/builder/svm-configure.jar \
--module \
org.graalvm.nativeimage.builder/com.oracle.svm.hosted.NativeImageGeneratorRunner \
-keepalive \
/proc/12565/comm \
-imagecp \
/home/runner/work/graalvm-reachability-metadata/graalvm-reachability-metadata/tests/src/org.apache.activemq/activemq-broker/6.0.0/build/libs/org.apache.activemq.activemq-broker_tests.jar:/home/runner/.gradle/caches/modules-2/files-2.1/org.graalvm.buildtools/junit-platform-native/1.0.0/2974f834a2f0a0105f2284fba7ef16dc4d60de56/junit-platform-native-1.0.0.jar:/home/runner/.gradle/caches/modules-2/files-2.1/org.junit.jupiter/junit-jupiter/5.9.2/26c586fbe0ebd81b48c9f11f0d998124248697ae/junit-jupiter-5.9.2.jar:/home/runner/.gradle/caches/modules-2/files-2.1/org.junit.platform/junit-platform-reporting/1.9.2/8bea292e2a546e4c21071f6bee60917ffd9f0101/junit-platform-reporting-1.9.2.jar:/home/runner/.gradle/caches/modules-2/files-2.1/org.junit.platform/junit-platform-launcher/1.9.2/38e0bfad8c57d4cd1a8f27926c25ffe9543068d6/junit-platform-launcher-1.9.2.jar:/home/runner/.gradle/caches/modules-2/files-2.1/org.junit.vintage/junit-vintage-engine/5.9.2/53421816bde124a564a64ba005dcc0c8e66a9722/junit-vintage-engine-5.9.2.jar:/home/runner/.gradle/caches/modules-2/files-2.1/org.junit.jupiter/junit-jupiter-params/5.9.2/bc2765afb7b85b583c710dd259a11c6b8c39e912/junit-jupiter-params-5.9.2.jar:/home/runner/.gradle/caches/modules-2/files-2.1/org.junit.jupiter/junit-jupiter-engine/5.9.2/572f7a553b53f83ee59cc045ce1c3772864ab76c/junit-jupiter-engine-5.9.2.jar:/home/runner/.gradle/caches/modules-2/files-2.1/org.junit.jupiter/junit-jupiter-api/5.9.2/fed843581520eac594bc36bb4b0f55e7b947dda9/junit-jupiter-api-5.9.2.jar:/home/runner/.gradle/caches/modules-2/files-2.1/org.junit.platform/junit-platform-engine/1.9.2/40aeef2be7b04f96bb91e8b054affc28b7c7c935/junit-platform-engine-1.9.2.jar:/home/runner/.gradle/caches/modules-2/files-2.1/org.junit.platform/junit-platform-commons/1.9.2/6f9f8621d8230cd38aa42e58ccbc0c00569131ce/junit-platform-commons-1.9.2.jar:/home/runner/.gradle/caches/modules-2/files-2.1/org.junit.platform/junit-platform-console/1.9.2/5548148dd55e820c42bd817ab93c1eaa05ce37f6/junit-platform-console-1.9.2.jar:/home/runner/work/graalvm-reachability-metadata/graalvm-reachability-metadata/tests/tck-test-support/build/libs/tck-test-support-1.0.0-SNAPSHOT.jar:/home/runner/.gradle/caches/modules-2/files-2.1/org.apache.activemq/activemq-broker/6.0.0/9e02abbcd317f4c0b22c96d29f63da2e0f7b9806/activemq-broker-6.0.0.jar:/home/runner/.gradle/caches/modules-2/files-2.1/org.assertj/assertj-core/3.22.0/c300c0c6a24559f35fa0bd3a5472dc1edcd0111e/assertj-core-3.22.0.jar:/home/runner/.gradle/caches/modules-2/files-2.1/org.apache.activemq/activemq-openwire-legacy/6.0.0/817a51aaaef75076d39aae18e5fff6f63f1e3c3b/activemq-openwire-legacy-6.0.0.jar:/home/runner/.gradle/caches/modules-2/files-2.1/org.apache.activemq/activemq-client/6.0.0/b4196c428e00601d9dd8a4c347a7117618a805eb/activemq-client-6.0.0.jar:/home/runner/.gradle/caches/modules-2/files-2.1/jakarta.annotation/jakarta.annotation-api/2.1.1/48b9bda22b091b1f48b13af03fe36db3be6e1ae3/jakarta.annotation-api-2.1.1.jar:/home/runner/.gradle/caches/modules-2/files-2.1/com.fasterxml.jackson.core/jackson-annotations/2.15.3/79baf4e605eb3bbb60b1c475d44a7aecceea1d60/jackson-annotations-2.15.3.jar:/home/runner/.gradle/caches/modules-2/files-2.1/com.fasterxml.jackson.core/jackson-core/2.15.3/60d600567c1862840397bf9ff5a92398edc5797b/jackson-core-2.15.3.jar:/home/runner/.gradle/caches/modules-2/files-2.1/com.fasterxml.jackson.core/jackson-databind/2.15.3/a734bc2c47a9453c4efa772461a3aeb273c010d9/jackson-databind-2.15.3.jar:/home/runner/.gradle/caches/modules-2/files-2.1/junit/junit/4.13.2/8ac9e16d933b6fb43bc7f576336b8f4d7eb5ba12/junit-4.13.2.jar:/home/runner/.gradle/caches/modules-2/files-2.1/org.slf4j/slf4j-api/2.0.9/7cf2726fdcfbc8610f9a71fb3ed639871f315340/slf4j-api-2.0.9.jar:/home/runner/.gradle/caches/modules-2/files-2.1/jakarta.jms/jakarta.jms-api/3.1.0/e194cf91a3f908e4846542849ac11a8e0b3c68ad/jakarta.jms-api-3.1.0.jar:/home/runner/.gradle/caches/modules-2/files-2.1/org.fusesource.hawtbuf/hawtbuf/1.11/8f0e50ad8bea37b84b698ec40cce09e47714a63e/hawtbuf-1.11.jar:/home/runner/.gradle/caches/modules-2/files-2.1/org.opentest4j/opentest4j/1.2.0/28c11eb91f9b6d8e200631d46e20a7f407f2a046/opentest4j-1.2.0.jar:/home/runner/.gradle/caches/modules-2/files-2.1/org.hamcrest/hamcrest-core/1.3/42a25dc3219429f0e5d060061f71acb49bf010a0/hamcrest-core-1.3.jar:/home/runner/work/graalvm-reachability-metadata/graalvm-reachability-metadata/tests/src/org.apache.activemq/activemq-broker/6.0.0/build/classes/java/test:/home/runner/work/graalvm-reachability-metadata/graalvm-reachability-metadata/tests/src/org.apache.activemq/activemq-broker/6.0.0/build/resources/test:/home/runner/work/graalvm-reachability-metadata/graalvm-reachability-metadata/tests/src/org.apache.activemq/activemq-broker/6.0.0/build/test-results/test/testlist \
-imagemp \
/opt/hostedtoolcache/graalvm-jdk-latest-ea_linux-x64_bin/25.0.0/x64/graalvm-25.1.0-dev+10.1/lib/svm/library-support.jar:/opt/hostedtoolcache/graalvm-jdk-latest-ea_linux-x64_bin/25.0.0/x64/graalvm-25.1.0-dev+10.1/lib/svm/svm-guest.jar \
-H:CLibraryPath=/opt/hostedtoolcache/graalvm-jdk-latest-ea_linux-x64_bin/25.0.0/x64/graalvm-25.1.0-dev+10.1/lib/svm/clibraries/linux-amd64/glibc,/opt/hostedtoolcache/graalvm-jdk-latest-ea_linux-x64_bin/25.0.0/x64/graalvm-25.1.0-dev+10.1/lib/svm/clibraries/linux-amd64,/opt/hostedtoolcache/graalvm-jdk-latest-ea_linux-x64_bin/25.0.0/x64/graalvm-25.1.0-dev+10.1/lib/svm/clibraries \
-H:Path@driver=/home/runner/work/graalvm-reachability-metadata/graalvm-reachability-metadata/tests/src/org.apache.activemq/activemq-broker/6.0.0/build/native/nativeTestCompile \
-H:BuildOutputJSONFile@user=/tmp/native-image-build-output.json \
-H:FallbackThreshold@user+api=0 \
-H:Name@user+api=org.apache.activemq.activemq-broker_tests-tests \
-H:ConfigurationFileDirectories@user=/home/runner/work/graalvm-reachability-metadata/graalvm-reachability-metadata/tests/src/org.apache.activemq/activemq-broker/6.0.0/build/native/generated/generateTestResourcesConfigFile,/home/runner/work/graalvm-reachability-metadata/graalvm-reachability-metadata/metadata/org.opentest4j/opentest4j/1.2.0,/home/runner/work/graalvm-reachability-metadata/graalvm-reachability-metadata/metadata/org.junit.platform/junit-platform-commons/1.8.2,/home/runner/work/graalvm-reachability-metadata/graalvm-reachability-metadata/metadata/org.junit.jupiter/junit-jupiter-params/5.9.2,/home/runner/work/graalvm-reachability-metadata/graalvm-reachability-metadata/metadata/org.junit.jupiter/junit-jupiter-engine/5.8.2,/home/runner/work/graalvm-reachability-metadata/graalvm-reachability-metadata/metadata/org.junit.platform/junit-platform-engine/1.10.4,/home/runner/work/graalvm-reachability-metadata/graalvm-reachability-metadata/metadata/org.junit.platform/junit-platform-launcher/1.9.2,/home/runner/work/graalvm-reachability-metadata/graalvm-reachability-metadata/metadata/org.junit.platform/junit-platform-reporting/1.9.2,/home/runner/work/graalvm-reachability-metadata/graalvm-reachability-metadata/metadata/org.junit.vintage/junit-vintage-engine/5.9.1,/home/runner/work/graalvm-reachability-metadata/graalvm-reachability-metadata/metadata/junit/junit/4.13.2,/home/runner/work/graalvm-reachability-metadata/graalvm-reachability-metadata/metadata/org.hamcrest/hamcrest-core/1.3,/home/runner/work/graalvm-reachability-metadata/graalvm-reachability-metadata/metadata/org.apache.activemq/activemq-broker/6.0.0,/home/runner/work/graalvm-reachability-metadata/graalvm-reachability-metadata/metadata/org.apache.activemq/activemq-client/6.0.0,/home/runner/work/graalvm-reachability-metadata/graalvm-reachability-metadata/metadata/org.slf4j/slf4j-api/1.7.36,/home/runner/work/graalvm-reachability-metadata/graalvm-reachability-metadata/metadata/jakarta.jms/jakarta.jms-api/3.0.0,/home/runner/work/graalvm-reachability-metadata/graalvm-reachability-metadata/metadata/org.fusesource.hawtbuf/hawtbuf/1.11,/home/runner/work/graalvm-reachability-metadata/graalvm-reachability-metadata/metadata/org.apache.activemq/activemq-openwire-legacy/5.18.1,/home/runner/work/graalvm-reachability-metadata/graalvm-reachability-metadata/metadata/jakarta.annotation/jakarta.annotation-api/2.1.0,/home/runner/work/graalvm-reachability-metadata/graalvm-reachability-metadata/metadata/com.fasterxml.jackson.core/jackson-databind/2.15.2,/home/runner/work/graalvm-reachability-metadata/graalvm-reachability-metadata/metadata/com.fasterxml.jackson.core/jackson-annotations/2.3.0,/home/runner/work/graalvm-reachability-metadata/graalvm-reachability-metadata/metadata/com.fasterxml.jackson.core/jackson-core/2.6.0 \
-H:Optimize@user+api=b \
-H:+ReportExceptionStackTraces@user \
-H:FutureDefaults@user+api=all \
-H:Features@user+api=org.graalvm.junit.platform.JUnitPlatformFeature \
'-H:Class@explicit main-class=org.graalvm.junit.platform.NativeImageJUnitLauncher' \
-H:ImageBuildID@driver=da0521ec-3271-f1a8-12ce-20f71be772c7 \
'-H:Features@jar:file:///opt/hostedtoolcache/graalvm-jdk-latest-ea_linux-x64_bin/25.0.0/x64/graalvm-25.1.0-dev+10.1/lib/svm/library-support.jar!/META-INF/native-image/com.oracle.svm/thirdparty/native-image.properties+api=com.oracle.svm.thirdparty.gson.GsonFeature' \
'-H:Features@jar:file:///opt/hostedtoolcache/graalvm-jdk-latest-ea_linux-x64_bin/25.0.0/x64/graalvm-25.1.0-dev+10.1/lib/svm/library-support.jar!/META-INF/native-image/com.oracle.svm/polyglot/native-image.properties+api=com.oracle.svm.polyglot.groovy.GroovyIndyInterfaceFeature' \
'-H:AddExports@jar:file:///opt/hostedtoolcache/graalvm-jdk-latest-ea_linux-x64_bin/25.0.0/x64/graalvm-25.1.0-dev+10.1/lib/svm/svm-guest.jar!/META-INF/native-image/org.graalvm.nativeimage.guest/native-image.properties+api=java.base/jdk.internal.misc=org.graalvm.nativeimage.guest' \
'-H:Features@jar:file:///opt/hostedtoolcache/graalvm-jdk-latest-ea_linux-x64_bin/25.0.0/x64/graalvm-25.1.0-dev+10.1/lib/svm/svm-guest.jar!/META-INF/native-image/org.graalvm.nativeimage.guest/native-image.properties+api=com.oracle.svm.guest.features.jdk.hosted.JavaNetHttpFeature' \
'-H:ClassInitialization@jar:file:///opt/hostedtoolcache/graalvm-jdk-latest-ea_linux-x64_bin/25.0.0/x64/graalvm-25.1.0-dev+10.1/lib/svm/svm-guest.jar!/META-INF/native-image/org.graalvm.nativeimage.guest/native-image.properties+api=sun.net.httpserver.simpleserver:run_time' \
'-H:ReachabilityMetadataResources@jar:file:///opt/hostedtoolcache/graalvm-jdk-latest-ea_linux-x64_bin/25.0.0/x64/graalvm-25.1.0-dev+10.1/lib/svm/svm-guest.jar!/META-INF/native-image/org.graalvm.nativeimage.guest/reachability-metadata.json+api=META-INF/native-image/org.graalvm.nativeimage.guest/reachability-metadata.json' \
-H:DriverWarningsCount=4
]
Warning: Option 'FallbackThreshold' is deprecated and might be removed in a future release: It no longer has any effect, and no replacement is available. Please refer to the GraalVM release notes.
========================================================================================================================
GraalVM Native Image: Generating 'org.apache.activemq.activemq-broker_tests-tests' (executable)...
========================================================================================================================
For detailed information and explanations on the build output, visit:
https://github.com/oracle/graal/blob/master/docs/reference-manual/native-image/BuildOutput.md
------------------------------------------------------------------------------------------------------------------------
[1/8] Initializing...                                                                                   (7.3s @ 0.22GiB)
 Builder configuration:
 - Java version: 25.0.2+10-LTS, vendor version: Oracle GraalVM 25.1.0-dev+10.1
 - Graal compiler: optimization level: b, target machine: x86-64-v3, PGO: off
 - C compiler: gcc (linux, x86_64, 13.3.0)
 - Assertions: disabled, system assertions: disabled
 - 1 user-specific feature(s):
   - org.graalvm.junit.platform.JUnitPlatformFeature
 Image configuration:
 - Garbage collector: Serial GC (max heap size: 80% of RAM)
 - Assertions: disabled (class-specific config may apply), system assertions: disabled
------------------------------------------------------------------------------------------------------------------------
 1 experimental option(s) unlocked:
 - '-H:BuildOutputJSONFile' (origin(s): command line)
------------------------------------------------------------------------------------------------------------------------
Build resources:
 - 13.27GiB of memory (85% of system memory because $CI set to 'true')
 - 4 thread(s) (100.0% of 4 available processor(s), determined at start)
[junit-platform-native] Running in 'test listener' mode using files matching pattern [junit-platform-unique-ids*] found in folder [/home/runner/work/graalvm-reachability-metadata/graalvm-reachability-metadata/tests/src/org.apache.activemq/activemq-broker/6.0.0/build/test-results/test/testlist] and its subfolders.
[2/8] Performing analysis...  [*****]                                                                  (44.6s @ 1.15GiB)
   10,816 types,  13,008 fields, and  54,824 methods found reachable
      478 types,      89 fields, and   1,016 methods registered for reflection
       63 types,      71 fields, and      55 methods registered for JNI access
      159 resource accesses registered with 266.55KiB total size
        4 native libraries: dl, pthread, rt, z
[3/8] Building universe...                                                                              (6.3s @ 1.11GiB)
[4/8] Parsing methods...      [**]                                                                      (4.2s @ 1.19GiB)
[5/8] Inlining methods...     [***]                                                                     (2.6s @ 1.28GiB)
[6/8] Compiling methods...    [*****]                                                                  (26.4s @ 1.03GiB)
[7/8] Laying out methods...   [**]                                                                      (4.0s @ 1.30GiB)
[8/8] Creating image...       [**]                                                                      (4.2s @ 1.51GiB)
  13.76MiB (37.55%) for code area:    34,614 compilation units
  17.81MiB (48.60%) for image heap:  286,943 objects and 29 resources
   5.08MiB (13.85%) for other data
  36.65MiB in total image size, 33.75MiB in total file size
------------------------------------------------------------------------------------------------------------------------
Top 10 origins of code area:                                Top 10 object types in image heap:
   1.31MiB o.g.n.runtime/com.oracle.svm.core.code              4.32MiB byte[] for code metadata
 748.45KiB java.base/java.util                                 3.23MiB byte[] for string data
 408.29KiB java.base/jdk.internal.classfile.impl               1.72MiB java.base/java.lang.String
 349.29KiB java.base/java.lang                                 1.34MiB java.base/java.lang.Class
 325.64KiB org.apache.activemq.broker.region                   1.16MiB o.g.n.runtime/c.o.s.core.hub.DynamicHubCompanion
 276.11KiB java.base/java.util.concurrent                    945.95KiB byte[] for general heap data
 267.01KiB java.base/com.sun.crypto.provider                 551.38KiB byte[] for reflection metadata
 245.64KiB java.base/java.io                                 420.04KiB o.g.n.runtime/c.o.s.c.h.ImageReflectionMetadata
 231.93KiB java.base/java.text                               370.58KiB java.base/java.lang.Object[]
 223.43KiB org.assertj.core.internal.bytebuddy.jar.asm       329.97KiB java.base/j.u.concurrent.ConcurrentHashMap$Node
   9.45MiB for 412 more packages                               3.49MiB for 3002 more object types
                            Use '--emit build-report' to create a report with more details.
------------------------------------------------------------------------------------------------------------------------
Security report:
 - Binary includes Java deserialization.
 - CycloneDX SBOM with 24 component(s) is embedded in binary (1.26KiB). 1 type(s) could not be associated to a component.
 - Advanced obfuscation not enabled; enable with '-H:AdvancedObfuscation=""' (experimental support).
------------------------------------------------------------------------------------------------------------------------
Recommendations:
 G1GC: Use the G1 GC ('--gc=G1') for improved latency and throughput.
 PGO:  Use Profile-Guided Optimizations ('--pgo') for improved throughput.
 HEAP: Set max heap for improved and more predictable memory usage.
 CPU:  Enable more CPU features with '-march=native' for improved performance.
------------------------------------------------------------------------------------------------------------------------
                       7.9s (7.6% of total time) in 1036 GCs | Peak RSS: 2.18GiB | CPU load: 3.63
------------------------------------------------------------------------------------------------------------------------
Build artifacts:
 /home/runner/work/graalvm-reachability-metadata/graalvm-reachability-metadata/tests/src/org.apache.activemq/activemq-broker/6.0.0/build/native/nativeTestCompile/libmanagement_ext.so (jdk_library, 32.23KiB)
 /home/runner/work/graalvm-reachability-metadata/graalvm-reachability-metadata/tests/src/org.apache.activemq/activemq-broker/6.0.0/build/native/nativeTestCompile/org.apache.activemq.activemq-broker_tests-tests (executable, 33.75MiB)
 /tmp/native-image-build-output.json (build_info, 1.05KiB)
========================================================================================================================
Finished generating 'org.apache.activemq.activemq-broker_tests-tests' in 1m 42s.

The build process encountered 5 warnings.

[native-image-plugin] Native Image written to: /home/runner/work/graalvm-reachability-metadata/graalvm-reachability-metadata/tests/src/org.apache.activemq/activemq-broker/6.0.0/build/native/nativeTestCompile

> Task :nativeTest FAILED
[junit-platform-native] WARNING: -djunit.platform.listeners.uid.tracking.output.dir not specified, trying to find test-ids on default Gradle/Maven locations. As this is a fallback mode, it could take a while. This should only happen if you are running tests executable manually and you didn't pass uid output directory with -djunit.platform.listeners.uid.tracking.output.dir=<path>.
[junit-platform-native] WARNING: Using test-ids from default Gradle project location:/home/runner/work/graalvm-reachability-metadata/graalvm-reachability-metadata/tests/src/org.apache.activemq/activemq-broker/6.0.0/build/test-results/test/testlist
[junit-platform-native] Running in 'test listener' mode using files matching pattern [junit-platform-unique-ids*] found in folder [/home/runner/work/graalvm-reachability-metadata/graalvm-reachability-metadata/tests/src/org.apache.activemq/activemq-broker/6.0.0/build/test-results/test/testlist] and its subfolders.
JUnit Platform on Native Image - report
----------------------------------------

org_apache_activemq.activemq_broker.ActivemqBrokerTest > testEmbeddedBrokerConnection() FAILED


Failures (1):
  JUnit Jupiter:ActivemqBrokerTest:testEmbeddedBrokerConnection()
    MethodSource [className = 'org_apache_activemq.activemq_broker.ActivemqBrokerTest', methodName = 'testEmbeddedBrokerConnection', methodParameterTypes = '']
    => jakarta.jms.JMSException: Could not create Transport. Reason: java.lang.NullPointerException
       org.apache.activemq.util.JMSExceptionSupport.create(JMSExceptionSupport.java:49)
       org.apache.activemq.ActiveMQConnectionFactory.createTransport(ActiveMQConnectionFactory.java:378)
       org.apache.activemq.ActiveMQConnectionFactory.createActiveMQConnection(ActiveMQConnectionFactory.java:391)
       org.apache.activemq.ActiveMQConnectionFactory.createActiveMQConnection(ActiveMQConnectionFactory.java:349)
       org.apache.activemq.ActiveMQConnectionFactory.createConnection(ActiveMQConnectionFactory.java:245)
       [...]
     Caused by: java.lang.NullPointerException
       org.apache.activemq.broker.jmx.ManagementContext.registerMBean(ManagementContext.java:415)
       org.apache.activemq.broker.jmx.AnnotatedMBean.registerMBean(AnnotatedMBean.java:93)
       org.apache.activemq.broker.BrokerService.startManagementContext(BrokerService.java:2612)
       org.apache.activemq.broker.BrokerService.start(BrokerService.java:607)
       org.apache.activemq.transport.vm.VMTransportFactory.doCompositeConnect(VMTransportFactory.java:127)
       [...]

Test run finished after 104 ms
[         3 containers found      ]
[         0 containers skipped    ]
[         3 containers started    ]
[         0 containers aborted    ]
[         3 containers successful ]
[         0 containers failed     ]
[         1 tests found           ]
[         0 tests skipped         ]
[         1 tests started         ]
[         0 tests aborted         ]
[         0 tests successful      ]
[         1 tests failed          ]


[Incubating] Problems report is available at: file:///home/runner/work/graalvm-reachability-metadata/graalvm-reachability-metadata/tests/src/org.apache.activemq/activemq-broker/6.0.0/build/reports/problems/problems-report.html

Deprecated Gradle features were used in this build, making it incompatible with Gradle 10.

You can use '--warning-mode all' to show the individual deprecation warnings and determine if they come from your own scripts or plugins.

For more on this, please refer to https://docs.gradle.org/9.1.0/userguide/command_line_interface.html#sec:command_line_warnings in the Gradle documentation.
17 actionable tasks: 7 executed, 10 up-to-date
SLF4J: No SLF4J providers were found.
SLF4J: Defaulting to no-operation (NOP) logger implementation
SLF4J: See https://www.slf4j.org/codes.html#noProviders for further details.

FAILURE: Build failed with an exception.

* What went wrong:
Execution failed for task ':nativeTest'.
> Process 'command '/home/runner/work/graalvm-reachability-metadata/graalvm-reachability-metadata/tests/src/org.apache.activemq/activemq-broker/6.0.0/build/native/nativeTestCompile/org.apache.activemq.activemq-broker_tests-tests'' finished with non-zero exit value 1

* Try:
> Run with --stacktrace option to get the stack trace.
> Run with --info or --debug option to get more log output.
> Run with --scan to generate a Build Scan (Powered by Develocity).
> Get more help at https://help.gradle.org.

BUILD FAILED in 1m 46s

> Task :tckTest FAILED

[Incubating] Problems report is available at: file:///home/runner/work/graalvm-reachability-metadata/graalvm-reachability-metadata/build/reports/problems/problems-report.html

FAILURE: Build failed with an exception.

* What went wrong:
Execution failed for task ':tckTest'.
> Test for org.apache.activemq:activemq-broker:6.0.0 failed with exit code 1.

* Try:
> Run with --stacktrace option to get the stack trace.
> Run with --info or --debug option to get more log output.
> Run with --scan to generate a Build Scan (Powered by Develocity).
> Get more help at https://help.gradle.org.

Deprecated Gradle features were used in this build, making it incompatible with Gradle 10.

You can use '--warning-mode all' to show the individual deprecation warnings and determine if they come from your own scripts or plugins.

For more on this, please refer to https://docs.gradle.org/9.1.0/userguide/command_line_interface.html#sec:command_line_warnings in the Gradle documentation.

BUILD FAILED in 1m 47s
11 actionable tasks: 3 executed, 8 up-to-date
Finished stage 'native-image run' for 6.0.0 at 2026-05-12T10:38:58Z with exit code 1 after 108s
❌ ERROR: 'native-image run' for 6.0.0 failed with exit code 1.
FAILED[native-image run][6.0.0][GVM_TCK_NATIVE_IMAGE_MODE="future-defaults-all" GVM_TCK_LV="6.0.0" ./gradlew clean test -Pcoordinates="org.apache.activemq:activemq-broker:6.0.0"]
```

--- a/metadata/org.apache.activemq/activemq-broker/6.0.0/reachability-metadata.json
+++ b/metadata/org.apache.activemq/activemq-broker/6.0.0/reachability-metadata.json
@@ -2,6 +2,280 @@
   "reflection": [
     {
       "condition": {
+        "typeReached": "org.apache.activemq.broker.BrokerService"
+      },
+      "type": "boolean[]"
+    },
+    {
+      "condition": {
+        "typeReached": "org.apache.activemq.broker.BrokerService"
+      },
+      "type": "byte[]"
+    },
+    {
+      "condition": {
+        "typeReached": "org.apache.activemq.broker.BrokerService"
+      },
+      "type": "char[]"
+    },
+    {
+      "condition": {
+        "typeReached": "org.apache.activemq.broker.BrokerService"
+      },
+      "type": "double[]"
+    },
+    {
+      "condition": {
+        "typeReached": "org.apache.activemq.broker.jmx.ManagementContext"
+      },
+      "type": "com.oracle.svm.core.genscavenge.IncrementalGarbageCollectorMXBean"
+    },
+    {
+      "condition": {
+        "typeReached": "org.apache.activemq.broker.jmx.ManagementContext"
+      },
+      "type": "com.oracle.svm.core.jdk.management.SubstrateClassLoadingMXBean"
+    },
+    {
+      "condition": {
+        "typeReached": "org.apache.activemq.broker.jmx.ManagementContext"
+      },
+      "type": "com.oracle.svm.core.jdk.management.SubstrateCompilationMXBean"
+    },
+    {
+      "condition": {
+        "typeReached": "org.apache.activemq.broker.jmx.ManagementContext"
+      },
+      "type": "com.oracle.svm.core.jdk.management.SubstrateRuntimeMXBean"
+    },
+    {
+      "condition": {
+        "typeReached": "org.apache.activemq.broker.jmx.ManagementContext"
+      },
+      "type": "com.oracle.svm.core.jdk.management.SubstrateThreadMXBean"
+    },
+    {
+      "condition": {
+        "typeReached": "org.apache.activemq.broker.jmx.ManagementContext"
+      },
+      "type": "com.sun.jmx.mbeanserver.ClassLoaderRepositorySupport$LoaderEntry[]"
+    },
+    {
+      "condition": {
+        "typeReached": "org.apache.activemq.broker.jmx.ManagementContext"
+      },
+      "type": "com.sun.management.GarbageCollectorMXBean"
+    },
+    {
+      "condition": {
+        "typeReached": "org.apache.activemq.broker.jmx.ManagementContext"
+      },
+      "type": "com.sun.management.GcInfo"
+    },
+    {
+      "condition": {
+        "typeReached": "org.apache.activemq.broker.jmx.ManagementContext"
+      },
+      "type": "com.sun.management.HotSpotDiagnosticMXBean"
+    },
+    {
+      "condition": {
+        "typeReached": "org.apache.activemq.broker.jmx.ManagementContext"
+      },
+      "type": "com.sun.management.OperatingSystemMXBean"
+    },
+    {
+      "condition": {
+        "typeReached": "org.apache.activemq.broker.jmx.ManagementContext"
+      },
+      "type": "com.sun.management.ThreadMXBean"
+    },
+    {
+      "condition": {
+        "typeReached": "org.apache.activemq.broker.jmx.ManagementContext"
+      },
+      "type": "com.sun.management.UnixOperatingSystemMXBean"
+    },
+    {
+      "condition": {
+        "typeReached": "org.apache.activemq.broker.jmx.ManagementContext"
+      },
+      "type": "com.sun.management.VMOption"
+    },
+    {
+      "condition": {
+        "typeReached": "org.apache.activemq.broker.jmx.ManagementContext"
+      },
+      "type": "com.sun.management.internal.DiagnosticCommandArgumentInfo",
+      "jniAccessible": true,
+      "methods": [
+        {
+          "name": "<init>",
+          "parameterTypes": [
+            "java.lang.String",
+            "java.lang.String",
+            "java.lang.String",
+            "java.lang.String",
+            "boolean",
+            "boolean",
+            "boolean",
+            "int"
+          ]
+        }
+      ]
+    },
+    {
+      "condition": {
+        "typeReached": "org.apache.activemq.broker.jmx.ManagementContext"
+      },
+      "type": "com.sun.management.internal.DiagnosticCommandArgumentInfo[]",
+      "jniAccessible": true
+    },
+    {
+      "condition": {
+        "typeReached": "org.apache.activemq.broker.jmx.ManagementContext"
+      },
+      "type": "com.sun.management.internal.DiagnosticCommandInfo",
+      "jniAccessible": true,
+      "methods": [
+        {
+          "name": "<init>",
+          "parameterTypes": [
+            "java.lang.String",
+            "java.lang.String",
+            "java.lang.String",
+            "java.lang.String",
+            "java.lang.String",
+            "java.lang.String",
+            "boolean",
+            "java.util.List"
+          ]
+        }
+      ]
+    },
+    {
+      "condition": {
+        "typeReached": "org.apache.activemq.broker.jmx.ManagementContext"
+      },
+      "type": "com.sun.management.internal.DiagnosticCommandInfo[]",
+      "jniAccessible": true
+    },
+    {
+      "condition": {
+        "typeReached": "org.apache.activemq.broker.jmx.ManagementContext"
+      },
+      "type": "com.sun.management.internal.GarbageCollectorExtImpl"
+    },
+    {
+      "condition": {
+        "typeReached": "org.apache.activemq.broker.jmx.ManagementContext"
+      },
+      "type": "com.sun.management.internal.HotSpotDiagnostic"
+    },
+    {
+      "condition": {
+        "typeReached": "org.apache.activemq.broker.jmx.ManagementContext"
+      },
+      "type": "com.sun.management.internal.HotSpotThreadImpl"
+    },
+    {
+      "condition": {
+        "typeReached": "org.apache.activemq.broker.jmx.ManagementContext"
+      },
+      "type": "com.sun.management.internal.OperatingSystemImpl"
+    },
+    {
+      "condition": {
+        "typeReached": "org.apache.activemq.broker.BrokerService"
+      },
+      "type": "float[]"
+    },
+    {
+      "condition": {
+        "typeReached": "org.apache.activemq.broker.BrokerService"
+      },
+      "type": "int[]"
+    },
+    {
+      "condition": {
+        "typeReached": "org.apache.activemq.broker.BrokerService"
+      },
+      "type": "java.lang.String[]"
+    },
+    {
+      "condition": {
+        "typeReached": "org.apache.activemq.broker.jmx.ManagementContext"
+      },
+      "type": "java.lang.management.ManagementFactory",
+      "methods": [
+        {
+          "name": "getPlatformMBeanServer",
+          "parameterTypes": []
+        }
+      ]
+    },
+    {
+      "condition": {
+        "typeReached": "org.apache.activemq.broker.jmx.ManagementContext"
+      },
+      "type": "java.lang.StackTraceElement[]"
+    },
+    {
+      "condition": {
+        "typeReached": "org.apache.activemq.broker.jmx.ManagementContext"
+      },
+      "type": "java.lang.reflect.Method[]"
+    },
+    {
+      "condition": {
+        "typeReached": "org.apache.activemq.util.IntrospectionSupport"
+      },
+      "type": "java.lang.reflect.Method[]"
+    },
+    {
+      "condition": {
+        "typeReached": "org.apache.activemq.broker.jmx.ManagementContext"
+      },
+      "type": "java.util.Map"
+    },
+    {
+      "condition": {
+        "typeReached": "org.apache.activemq.broker.BrokerService"
+      },
+      "type": "org.bouncycastle.jce.provider.BouncyCastleProvider"
+    },
+    {
+      "condition": {
+        "typeReached": "org.apache.activemq.broker.BrokerService"
+      },
+      "type": "javax.management.openmbean.CompositeData[]"
+    },
+    {
+      "condition": {
+        "typeReached": "org.apache.activemq.broker.BrokerService"
+      },
+      "type": "long[]"
+    },
+    {
+      "condition": {
+        "typeReached": "org.apache.activemq.broker.BrokerService"
+      },
+      "type": "org.apache.activemq.Service",
+      "methods": [
+        {
+          "name": "start",
+          "parameterTypes": []
+        }
+      ]
+    },
+    {
+      "condition": {
+        "typeReached": "org.apache.activemq.broker.BrokerService"
+      },
+      "type": "org.apache.activemq.command.ActiveMQDestination[]"
+    },
+    {
+      "condition": {
         "typeReached": "org.apache.activemq.util.IntrospectionSupport"
       },
       "type": "org.apache.activemq.broker.BrokerService",
@@ -48,6 +322,30 @@
           "name": "lastSequenceId"
         }
       ]
+    },
+    {
+      "condition": {
+        "typeReached": "org.apache.activemq.util.IdGenerator"
+      },
+      "type": "jdk.net.LinuxSocketOptions"
+    },
+    {
+      "condition": {
+        "typeReached": "org.apache.activemq.broker.jmx.ManagementContext"
+      },
+      "type": "javax.management.MBeanAttributeInfo[]"
+    },
+    {
+      "condition": {
+        "typeReached": "org.apache.activemq.broker.jmx.ManagementContext"
+      },
+      "type": "javax.management.MBeanServerBuilder"
+    },
+    {
+      "condition": {
+        "typeReached": "org.apache.activemq.broker.BrokerService"
+      },
+      "type": "short[]"
     }
   ],
   "resources": [
@@ -62,6 +360,12 @@
         "typeReached": "org.apache.activemq.transport.TransportFactory"
       },
       "glob": "META-INF/services/org/apache/activemq/transport/vm"
+    },
+    {
+      "condition": {
+        "typeReached": "org.apache.activemq.broker.BrokerService"
+      },
+      "glob": "META-INF/services/org.slf4j.spi.SLF4JServiceProvider"
     },
     {
       "condition": {

--- a/stats/org.apache.activemq/activemq-broker/6.0.0/stats.json
+++ b/stats/org.apache.activemq/activemq-broker/6.0.0/stats.json
@@ -20,21 +20,21 @@
     },
     "libraryCoverage" : {
       "instruction" : {
-        "covered" : 8855,
-        "missed" : 95875,
-        "ratio" : 0.084551,
+        "covered" : 8892,
+        "missed" : 95838,
+        "ratio" : 0.084904,
         "total" : 104730
       },
       "line" : {
-        "covered" : 2309,
-        "missed" : 23927,
-        "ratio" : 0.088009,
+        "covered" : 2316,
+        "missed" : 23920,
+        "ratio" : 0.088276,
         "total" : 26236
       },
       "method" : {
-        "covered" : 533,
-        "missed" : 5599,
-        "ratio" : 0.086921,
+        "covered" : 534,
+        "missed" : 5598,
+        "ratio" : 0.087084,
         "total" : 6132
       }
     }

